control-plane-operator/hostedclusterconfigoperator/controllers/inplaceupgrader/inplaceupgrader.go

```diff
@@ -211,7 +211,7 @@ func (r *Reconciler) reconcileInPlaceUpgrade(ctx context.Context, nodePoolUpgrad
 	// Signal in-place upgrade progress.
 	result, err := r.CreateOrUpdate(ctx, r.client, machineSet, func() error {
 		delete(machineSet.Annotations, nodePoolAnnotationUpgradeInProgressFalse)
-		machineSet.Annotations[nodePoolAnnotationUpgradeInProgressTrue] = fmt.Sprintf("Updating version in progress. Target version: %q. Total Nodes: %d. Upgraded: %d", *machineSet.Spec.Template.Spec.Version, len(nodes), len(nodes)-nodeNeedUpgradeCount)
+		machineSet.Annotations[nodePoolAnnotationUpgradeInProgressTrue] = fmt.Sprintf("Nodepool update in progress. Target Config version: %s. Total Nodes: %d. Upgraded: %d", nodePoolUpgradeAPI.spec.targetConfigVersion, len(nodes), len(nodes)-nodeNeedUpgradeCount)
```

Comment:
Config version -> should be ConfigVersion hash, it's a (may be bad) name for the hash that contains both inputs.